CHANGELOG.md

```diff
@@ -1,5 +1,122 @@
 # Changelog
 
+## [3.1.0](https://github.com/googleapis/python-spanner-django/compare/v3.0.2...v3.1.0) (2024-04-23)
+
+
+### ⚠ BREAKING CHANGES
```

Comment:
Can we get rid of these changes as well?

Reply:
Done